tests/ReportbackApiTest.php

```diff
@@ -22,6 +23,18 @@ class ReportbackApiTest extends TestCase
      */
     public function testCreatingNewReportback()
     {
+        // Mock sending image to AWS.
+        $this->fileSystem->shouldReceive('put')->andReturn(true);
+
+        // Create an uploaded file.
+        $path = storage_path('images/huskycorgi.jpeg');
+        $original_name = 'huskycorgi.jpeg';
+        $mime_type = 'image/jpeg';
+        $size = 2476;
```

Comment:
Is this the size of the `huskycorgi.jpeg` image? Not sure if it'd make sense to infer from the path if so. Definitely not a blocker if tests are passing, but perhaps a good future improvement!